File: java/amp-eventing/src/main/java/amp/eventing/streaming/EndOfStream.java
package amp.eventing.streaming;

public class EndOfStream {
    private String streamType;
    private String sequenceId;

    public String getStreamType() {
        return streamType;
    }

    public void setStreamType(String streamType) {
        this.streamType = streamType;
    }

    public EndOfStream() {
        setStreamType(null);
    }

    public EndOfStream(String streamType, String sequenceId) {
        setStreamType(streamType);
        setSequenceId(sequenceId);
    }

    public String getSequenceId() {
        return sequenceId;
    }

    public void setSequenceId(String sequenceId) {
        this.sequenceId = sequenceId;
    }
}

File: dotnet/source/amp.eventing/streaming/EndOfStream.cs
namespace amp.eventing.streaming
{
    public class EndOfStream
    {
        public string StreamType { get; set; }
        public string SequenceId { get; set; }

        public EndOfStream()
        {
            StreamType = null;
            SequenceId = null;
        }

        public EndOfStream(string streamType, string sequenceId)
        {
            StreamType = streamType;
            SequenceId = sequenceId;
        }
    }
}
